src/api/types.ts

```diff
@@ -1,5 +1,4 @@
 export interface BidParamsDto {
```

Comment:
You modified the API endpoints but I don't see those changes reflected in the API interactions here.

Reply:
Some changes weren't in yet when you saw this possibly. Maybe check again, I tested several different functions and code paths against the API and they all were successful